pkg/kube/kube_handler.go

```diff
@@ -43,6 +46,22 @@ import (
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 )
 
+var (
+	maxReq = 20
```

Comment:
Is there a reason you want a package variable here requiring initialization in an `init()` function? There are a bunch of reasons to avoid doing so ([Dave Cheney's thought experiment of removing package vars](https://dave.cheney.net/2017/06/11/go-without-package-scoped-variables) also outlines some of the why's).

Normally the main reason to do something like this (set a package var, use an `init()` fn to reach into the env and set the state, is because it's quicker to code (ie. short-term benefit). The one place where we do this usually is in our `main.go` to parse command-line args (which could be refactored, but not relevant to this PR).

I've got a strong preference to *remove* any precedents of package vars in our codebase when we can, not add to them. I think in this case, that'd not be too hard? Without having tried, it looks like we can update `NewClusterConfig` so that `maxRequests` is an option, and move the setting of this arg to the `main.go` which has the benefit of being both a cmd-line arg that is also set-able via an env var like the others. Then I think we can just pass it through the `handler.Options` to the handler and to `NewClusterConfig`? With that, it's an explicit cmd-line arg, rather than a magic way to reach into the state of the depths of the private handler library.

Is that OK? If you've got a good reason to sprinkle the setting of state in the depths of the app like this, let me know.